packages/plugin-dev/src/browser/hosted-plugin-controller.ts

```diff
@@ -247,7 +247,7 @@ export class HostedPluginController implements FrontendApplicationContribution {
     }
 
     private getErrorMessage(error: Error): string {
-        return error.message.substring(error.message.indexOf(':') + 1);
+        return error?.message?.substring(error.message.indexOf(':') + 1) || '';
```

Comment:
`error` is currently non optional so why do we do:

```ts
error?.message?.substring(...)
```

Reply:
This change was required after debugging exceptions thrown in some cases which didn't contain error or error.message which would result in hiding original stack trace of error.

Reply:
Change the type of the `error` parameter then, `any` could be enough. Right now it feels weird since `error` is supposed to always be defined, as well as `error.message`. But from what you are saying this is not always the case, so we should update the typings to reflect that.

Reply:
@marechal-p Fixed. Thanks